lightning-persister/src/fs_store.rs

```diff
@@ -470,7 +470,7 @@ mod tests {
 			index: 0
```

Comment:
10 lines up is a reference to permanent failure, though I suppose it's not really incorrect per se. Also in the test name.